src/components/DataCollector/utils.js

```diff
@@ -50,8 +50,23 @@ const isSupportedAnchorElement = element => {
   return false;
 };
 
+const isDownloadLink = (downloadLinkQualifier, linkUrl, clickedObj) => {
```

Comment:
Instead of passing the downloadLinkQualifier as a String and then creating a RegExp every time, you could create the RegExp once in createClickHandler, and then pass the RegExp object.

Reply:
I was thinking about doing that but added the extra code in the function instead. The performance impact will be minimal.